packages/stream_video/lib/src/core/client_state.dart

```diff
@@ -17,7 +20,7 @@ abstract class ClientState {
   StateEmitter<ConnectionState> get connection;
 
   /// Sets when a call being joined.
-  StateEmitter<Call?> get activeCall;
+  StateEmitter<List<Call>> get activeCalls;
```

Comment:
To prevent breaking changes we could keep the old one as well, but deprecate it.

I don't think this is the right place for this logic, but something similar like this:

I'm also not sure if we even need to deprecate it. Maybe just a nice feature anyway for apps that never have multiple active calls.

```dart
  /// Returns first of active calls or null if no calls are active
  @Deprecated("please use `activeCalls`")
  StateEmitter<Call?> get activeCall => activeCalls.firstOrNull;
  
  /// List of currently active calls.
  StateEmitter<List<Call>> get activeCalls;
```